tests/testutils/db_1.py

```diff
@@ -0,0 +1,73 @@
+"""db.py.
+
+Database level utility class for testing against timed_release model.
+"""
+
+from functools import wraps
+import sys
+
+from timed_release import config
+from timed_release.connectors.sql import db_session_maker
+
+
+DROP_TABLE_PRODUCT_LIVE_TIME = """
+DROP TABLE IF EXISTS asset;
+"""
+
+
+CREATE_TABLE_PRODUCT_LIVE_TIME = """
+CREATE TABLE product_live_time (
+  product_id int(10) PRIMARY KEY,
+  time_of_day_product varchar(200)),
+  time_zone varchar(200),
+  store_id int(10);
+"""
+
+
+def drop_product_live_time_table():
+    """DROP command for product_live_time table."""
+    _exit_if_not_test_environment(db_session_maker())
+    db_session_maker().execute(DROP_TABLE_PRODUCT_LIVE_TIME)
```

Comment:
i guess you should not use db_session_maker for this execution. Use with db_session() as session wala code